package.json

```diff
@@ -64,6 +64,7 @@
     "serverless-finch": "^4.0.3",
     "serverless-offline": "^13.2.0",
     "serverless-s3-local": "^0.8.1",
+    "uuid": "^10.0.0",
```

Comment:
This `uuid` package is not needed. Wherever you are using UUID, replace it with `crypto.randomUUID()`